components/recipes/modules/ROOT/partials/tools.adoc

```diff
@@ -0,0 +1,10 @@
+== Select your tool
+
+Ready to start? Below is a list of tools that this recipe may support.
+Find a tool that looks interesting and scroll down and pick your tool from the list of tabs. Don't see your tool for this recipe? Head over to the https://skale.chat[Discord] and request your preferred tool.
```

Comment:
https://discord.gg/skale

Reply:
Resolved in https://github.com/skalenetwork/docs.skale.network/pull/72/commits/cd434cb46eba715334b458c572dd66a3cb80881e